view/frontend/web/js/view/cart/apple-pay.js

```diff
@@ -0,0 +1,357 @@
+define([
+    'Magento_Checkout/js/view/payment/default',
+    'Magento_Checkout/js/model/quote',
+    'CardknoxDevelopment_Cardknox/js/view/cart/cardknox-apple-pay',
+    'ifields',
+    'Magento_Checkout/js/model/payment/additional-validators',
+    "jquery",
+    "ko",
+    'Magento_Checkout/js/model/full-screen-loader',
+    'Magento_Checkout/js/action/redirect-on-success',
+    'Magento_Checkout/js/action/place-order',
+    'Magento_Customer/js/model/customer',
+    'mage/url',
+    'Magento_Checkout/js/action/create-shipping-address',
+    'Magento_Checkout/js/action/create-billing-address',
+    'Magento_Checkout/js/model/shipping-save-processor/default'
+], function (
+    Component,
+    quote,
+    cardknoxApplePay,
+    ifields,
+    additionalValidators,
+    $,
+    koForAP,
+    fullScreenLoaderAP,
+    redirectOnSuccessActionAP,
+    placeOrderActionAP,
+    customer,
+    urlBuilder,
+    createShippingAddress,
+    createBillingAddress,
+    saveShipping
+) {
+    'use strict';
+    window.checkoutConfig.reloadOnBillingAddress = true;
+    const METHOD_ID = 'cardknox_apple_pay';
+
+    return Component.extend({
+        defaults: {
+            // template: 'CardknoxDevelopment_Cardknox/cart/apple-pay-btn',
```

Comment:
Is this comment needed?

Reply:
@mstein-ck, it's not needed now, we can remove it.

Reply:
Ok, can you remove it?

Reply:
@mstein-ck, I think you removed the comment code, I can't see it now.